PRE-COMMIT-README.md

```
@@ -0,0 +1,63 @@
+## Continuous Integration
+
+Some hooks (environment checks, Django collectstatic, and tests) are skipped in CI environments to avoid redundancy with existing CI pipelines. The configuration checks for the `GITHUB_ACTIONS` environment variable to determine the execution context.
+
+## Configuration
+
+The authoritative configuration for all pre-commit hooks is defined in [.pre-commit-config.yaml](pre-commit-config).
+
+## Manual Runs
+
+To manually run all pre-commit hooks:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)